xapian-letor/include/xapian-letor/scorer.h

```diff
@@ -89,11 +89,7 @@ class XAPIAN_VISIBILITY_DEFAULT ERRScore : public Scorer {
     /// Default constructor
     ERRScore();
 
-    /// Function to set max_grade
-    void set_max_grade(int max_grade_);
-
-    /// Function that returns the value set for max_grade.
-    int get_max_grade();
+    ERRScore(int max_grade_);
```

Comment:
You can default this argument so you only need one constructor; you can use an initializer list to set the `max_grade` member here (which at present will mean you don't need the constructor in the `.cc` file); and you must mark single-argument constructors `explicit` to avoid implicit conversion problems.